samples/python/odometry.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+
+import numpy as np
+import cv2 as cv
```

Comment:
Please add Python tests too for regular validation (`modules/3d/misc/python/test`)

Reply:
done